pkgs/applications/audio/soundtracker/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv
+, fetchurl
+, pkg-config
+, autoconf
+, gtk2
+, alsaLib
+, SDL
+, jack2
+, goocanvas # graphical envelope editing
+}:
+
+with stdenv.lib; stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation rec {
```

---

Did you test this? I'm using `optional` from `stdenv.lib`

---

Tested without this change. This should work (untested):

```suggestion
let inherit (stdenv.lib) optionals; in

stdenv.mkDerivation rec {
```

---

In the end I just qualified `optional`, since I'm only using it one time